ui/src/pages/backoffice/DocumentRequest/DocumentRequestDetails/components/DocumentInfo/DocumentInfo.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { Button, Grid, Header, Message, Segment } from 'semantic-ui-react';
+import { BackOfficeRoutes } from '@routes/urls';
+import { documentRequest as documentRequestApi } from '@api';
+import _get from 'lodash/get';
+import { DocumentIcon } from '@pages/backoffice/components';
+
+export default class DocumentInfo extends Component {
```

Comment:
I would just change a name of this component, because it is not info only, maybe `DocumentManager` or `DocumentOrganiser`